mathics/builtin/comparison.py

```diff
@@ -599,15 +594,7 @@ class Equal(_EqualityOperator, _SympyComparison):
     <dl>
       <dt>'Equal[$x$, $y$]'
       <dt>'$x$ == $y$'
-      <dd>is 'True' if $x$ and $y$ are known to be equal, or
-        'False' if $x$ and $y$ are known to be unequal, in which case
-        case, 'Not[$x$ == $y$]' will be 'True'.
-
```

Comment:
These line breaks are deliberate.

Reply:
Why here we can have here the line breaks?

Reply:
I may have mistagged  lines. I am seeing  _paragraph_  spacing around

"For any expression $x$ .." 

And "Commutative properties apply so"   (there should be a "," after apply). 

And in the replaced text I am not seeing that paragraph breaks are preserved.

Reply:
I was looking at this case. The problem here is that I expected that the `<dd>` tag contains a single paragraph, which follows the way in which (I think) WR is written. I proposed a change here to move the extra paragraphs after the `<dl>...</dl>` block.